google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/SpannerPoolTest.java

```diff
@@ -145,6 +151,9 @@ public void testGetSpanner() {
     spanner1 = pool.getSpanner(options3, connection1);
     spanner2 = pool.getSpanner(options4, connection2);
     assertThat(spanner1).isNotEqualTo(spanner2);
+    spanner1 = pool.getSpanner(options7, connection1);
```

Comment:
We are defining options4 and options5 below but using options7 and options8 here. Is it expected ?

Reply:
That is a different test (`testSpannerPoolKeyEquality`). This test is using the data members of the `SpannerPoolTest` class (`private ConnectionOptions options7 = mock(ConnectionOptions.class);`)